core/templates/dev/head/components/forms/schema_editors/SchemaBasedUnicodeEditorDirective.js

```diff
@@ -115,6 +116,22 @@ oppia.directive('schemaBasedUnicodeEditor', [
             return $sce.trustAsHtml(
               $filter('convertUnicodeWithParamsToHtml')($scope.localValue));
           };
+
+          var textBoxSelectedCounter = 0;
+          $scope.onInputFocus = function() {
+            textBoxSelectedCounter += 1;
+            if ($scope.helperMessageIsShown) {
+              $scope.helperMessageIsShown = false;
+            }
+          };
+
+          $scope.helperMessageIsShown = false;
+          $timeout(function() {
+            if (deviceInfoService.hasTouchEvents() && (
+                textBoxSelectedCounter == 1)) {
```

Comment:
This is because the text box has focus when the page loads. So, this counter becomes 1. We need to check if this number increases for us to know if the text box has been selected by the user. If the text box is always in focus, does the virtual keyboard show up instantly?